src/conversion/azure/contracts.ts

```diff
@@ -1,6 +1,6 @@
+import {IEntityStore, NodeSpec} from '../..';
```

Comment:
We might consider another convention for naming this sort of file. The term `contract` is used in the static analysis world to mean something else. Scenarios we should consider include

- io-ts type/schema definitions
- interfaces intended more as abstract base classes than POJOs.
- base classes